specificatie/BRK-Bevragen/genereervariant/openapi.yaml

```diff
@@ -3812,7 +3818,7 @@ components:
         einddatum:
           type: string
           format: date
-        einddatumRecht:
+        eindatumRecht:
```

Comment:
Moet dit niet met 2 d's zijn? Dus einddatumRecht?

Reply:
Dit zit dus al fout in de niet-resolved versie. En zat er al vanaf het begin fout in. Ergens is dus een correctie gedaan in de resolved versie (met correct 2 d's), maar niet in de niet-resolved versie.
Moet denk ik in beide gecorrigeerd worden.